_includes/scripts.html

```diff
@@ -40,21 +45,37 @@ <h5>Popular tags: </h5>
            <% if (project.tags) { %>
            <p class="tags">
                <% _.each(project.tags, function(tag, i) { %>
-                   <a href="#/tags/<%-encodeURIComponent(tag)%>"><%-tag%></a><%- i != project.tags.length-1 ? "," : "" %>
+                   <a href="#/page/1/tags/<%-encodeURIComponent(tag)%>"><%-tag%></a><%- i != project.tags.length-1 ? "," : "" %>
                <% }) %>
            </p>
-           <% } %>
+        <% }} %>
        </td>
     </tr>
     </tbody>
    <% }) %>
    </table>
+   <div class="center">
+     <div class="pagination">
+      <% var lastPage = parseInt((projects.length - 1) / nbrOfProjectsPerPage) + 1%>
+      <a id="previous" href="#/page/<%=currentPageNumber-1%>/tags/<%-encodeURIComponent(tagsString)%>" <% if(currentPageNumber <= 1) { %> class="not-active" href="#/page/<%=currentPageNumber%>/tags/<%-encodeURIComponent(tagsString)%>" <% } %> >&laquo; Prev</a>
+      <% _.each(_.range(1, projects.length + 1, nbrOfProjectsPerPage), function(i) {
+        var pageNumber = parseInt(i/nbrOfProjectsPerPage)+1;
+        if(pageNumber == 1 || pageNumber == lastPage || pageNumber == currentPageNumber) { %>
+        <a <%if(pageNumber == currentPageNumber) { %>
+          id="currentPage"
+          <% } %>
+          href="#/page/<%=pageNumber%>/tags/<%-encodeURIComponent(tagsString)%>"><%=pageNumber%></a>
+      <% }}) %>
+      <a id="next" href="#/page/<%=currentPageNumber-(-1)%>/tags/<%-encodeURIComponent(tagsString)%>" <% if(currentPageNumber >= lastPage) { %> class="not-active" <% } %>>Next &raquo;</a>
```

Comment:
haha yes I would believe `currentPageNumber+1` would work 😄

Reply:
actually after doing some digging it appears that `currentPageNumber+1` will actually evaluate to a string. i.e. if `currentPageNumber = '1'` then it will evaluate `currentPageNumber + 1` to `'11'`, but `currentPageNumber-(-1)` will evaluate to `2`.

Reply:
i believe the correct thing to do would be `parseInt(currentPageNumber)+1`